mono/mini/mini-runtime.c

```diff
@@ -104,6 +104,7 @@
 
 static guint32 default_opt = 0;
 static gboolean default_opt_set = FALSE;
+char *stats_method_name = NULL;
```

Comment:
I'd at least have a newline between this and the statics; they don't seem to be related.